src/main/scala/jp/co/septeni_original/k8sop/ConfigMapClient.scala

```diff
@@ -38,7 +38,7 @@ class ConfigMapClient(val client: ApiClient)(implicit ec: ExecutionContext) exte
     val cmF = ols2cmls(objects).map { (cm: V1ConfigMap) =>
       logger.debug(s"ConfigMap create start. cm: $cm")
       FutureOps.retryWithRefresh {
-        val res = api.createNamespacedConfigMap(cm.getMetadata.getNamespace, cm, "false")
+        val res = api.createNamespacedConfigMap(cm.getMetadata.getNamespace, cm, false, "false", null)
```

Comment:
The 6th parameter is for `dryRun` which is supported since k8s 1.13.

https://kubernetes.io/docs/reference/using-api/api-concepts/#dry-run

It's still beta and some k8s managed services(like GCP) doesn't support k8s 1.13 itself, so we don't pass it as the param(= null) at the moment.